runbotics/runbotics-orchestrator-ui/src/pages/index.tsx

```diff
@@ -26,9 +27,14 @@ export const getServerSideProps: GetServerSideProps<Props> = async ({ res, local
 
     const blogPosts = cache.posts?.slice(0, 3) ?? [];
 
+    const metadata: MetadataTags = {
+        image: HeroBg.src,
```

Comment:
doesn't it missing the title?

Reply:
no, home page gets default metadata from Metadata component, but I added custom description content